code/modules/mob/living/carbon/xenomorph/castes/carrier/abilities_carrier.dm

```diff
@@ -337,3 +337,94 @@ GLOBAL_LIST_INIT(hugger_images_list,  list(
 	turret.ammo = GLOB.ammo_list[GLOB.hugger_to_ammo[X.selected_hugger_type]]
 	succeed_activate()
 	add_cooldown()
+
+// ***************************************
+// *********** Call of younger
+// ***************************************
+
+/datum/action/xeno_action/activable/call_younger
+	name = "Call of younger"
+	action_icon_state = "call_younger"
+	mechanics_text = "Appeals to the larva inside the Marine. The Marine loses his balance, and larva's progress accelerates."
+	ability_name = "call younger"
+	plasma_cost = 150
+	cooldown_timer = 20 SECONDS
+	keybinding_signals = list(
+		KEYBINDING_NORMAL = COMSIG_XENOABILITY_CALL_YOUNGER,
+	)
+
+/datum/action/xeno_action/activable/call_younger/on_cooldown_finish()
+	to_chat(owner, span_notice("We are ready to make another call"))
+	owner.playsound_local(owner, 'sound/effects/xeno_newlarva.ogg', 25, 0, 1)
+	return ..()
+
+/datum/action/xeno_action/activable/call_younger/proc/check_los(atom/A)
+	if(!line_of_sight(owner, A))
+		return FALSE
+	return TRUE
+
+/datum/action/xeno_action/activable/call_younger/can_use_ability(atom/A, silent, override_flags)
+	. = ..()
+	if(!.)
+		return
+
+	if(!ishuman(A))
+		if(!silent)
+			A.balloon_alert(owner, "It's not a human")
```

Comment:
Not going to force you but some of these could be shorter (ie here "Not human" is more consise